string tape_color
string LAB
bool isTemp
---
bool isUpdated
